ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.collection.js

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
Wondering if we still want to re-visit the abstraction of this logic in this PR that is almost identical between result-form, search-form-sharing as well as search-form components.

Reply:
✏️ I agree we should revisit this; might want to make a ticket for it